docs/dashboards/types-of-components.md

```diff
@@ -33,50 +33,50 @@ Also please check the [Custom Component](https://www.highcharts.com/docs/dashboa
 ### Highcharts Component
 The option to include a Highcharts chart in one of the components is available out of the box. Here is the set of files that need to be included to make the Highcharts component work.
 ```html
-    <script src="https://code.highcharts.com/dashboards/dashboards.js"></script>
-    <script src="https://code.highcharts.com/highcharts.js"></script>
-    <script src="https://code.highcharts.com/dashboards/modules/dashboards-plugin.js"></script>
+<script src="https://code.highcharts.com/dashboards/dashboards.js"></script>
```

Comment:
Why don't you remove those spaces in other `.md` files like `installation`,`edit-mode` etc